docs/ref/checks.txt

```diff
@@ -327,6 +327,8 @@ Related fields
 * **fields.E339**: ``<model>.<field name>`` is not a foreign key to ``<model>``.
 * **fields.E340**: The field's intermediary table ``<table name>`` clashes with
   the table name of ``<model>``/``<model>.<field name>``.
+* **fields.E341**: ``<model>``.``related_name`` must be different than the
+  ``<model>``'s manager name.
```

Comment:
This needs to match the error. If you update to my suggestion, this would be:
```suggestion
* **fields.E341**: Related name ``<related name>`` for ``<model>.<field name>``
  clashes with the name of a model manager.
```